types/base/api.d.ts

```diff
@@ -0,0 +1,42 @@
+type Part = {
+    name: string;
+    material: string;
+    color: string;
```

Comment:
There are optional parts when this probably might be optional. Check the codebase to understand if that's the case

Maybe ask @Benedito0 or @joaofsl for help on debugging this on RIPE White and take a look inside the SDK state for models with optional parts.

Reply:
I have tried importing an order with just part name and no material and no color, and with name and material but no color and both test cases failed. I think they are mandatory but I will check with @Benedito0  and @joaofsl  first.

Reply:
Okay just call it `partTriplet` then.